css/css-fonts/variations/variable-box-font.html

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html class="reftest-wait">
+<link rel="help" href="https://www.w3.org/TR/css-fonts-4/#font-rend-desc">
```

Comment:
Shouldn't this link to the *property* at https://www.w3.org/TR/css-fonts-4/#font-variation-settings-def rather than the *descriptor* section?